ui/widgets/nuxeo-date-picker.js

```diff
@@ -209,6 +222,7 @@ import { I18nBehavior } from '../nuxeo-i18n-behavior.js';
       this.$.date.set('i18n.cancel', this.i18n('command.cancel'));
       this.$.date.set('i18n.clear', this.i18n('command.clear'));
       this.$.date.set('i18n.today', this.i18n('today'));
+      this.$.date.set('i18n.firstDayOfWeek', this.firstDayOfWeek);
```

Comment:
We shouldn't need this anymore since it will be run by the observer when the default value of `firstDayOfWeek` is set.

Reply:
Indeed. With the refactor I forgot this one.
Anyway, we should be aware that we are not able to change date picker layout by setting this variable after the rendering. It seems to be some "limitation" from `vaadin-date-picker`.